src/_pytest/config/__init__.py

```diff
@@ -1073,9 +1073,12 @@ def _checkversion(self):
                     )
                 )
 
-    def _validatekeys(self):
+    def _validatekeys(self, args: Sequence[str]):
         for key in self._get_unknown_ini_keys():
-            sys.stderr.write("WARNING: unknown config ini key: {}\n".format(key))
+            message = "Unknown config ini key: {}\n".format(key)
```

Comment:
when validation is put until after plugin loading, it can use the normal config and doesnt have to use the hack for the args